BLOGPOST.md

```diff
@@ -0,0 +1,38 @@
+# Introducing the SAP Cloud SDK for AI (JavaScript/TypeScript) 🎉
+
+We are excited to announce the initial release of the [SAP Cloud SDK for AI](https://github.com/SAP/ai-sdk-js#readme)! It simplifies the integration of generative AI capabilities into your SAP Business Technology Platform (BTP) applications.
+
+The SDK allows you to leverage the SAP AI Core and SAP AI Launchpad, enabling capabilities like templating, grounding, data masking, and more. In this blog post, we’ll introduce you to the key packages and their features.
```

Comment:
Please link to help documents for SAP AI Core & Launchpad.
AI core -> https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/what-is-sap-ai-core